Source/Engine/UI/GUI/CanvasRootControl.cs

```diff
@@ -54,6 +54,38 @@ public override Control FocusedControl
             }
         }
 
+        /// <summary>
+        /// Min scale value, anything below will get scaled up to fid this size
+        /// </summary>
+        public Vector2 MinCustomScaler = Vector2.Zero;
+        /// <summary>
+        /// Max scale value, anything above will get scaled down to fit this size
+        /// </summary>
+        public Vector2 MaxCustomScaler = Vector2.Maximum;
+        /// <summary>
+        /// Helper value to determine the custom scale;
+        /// </summary>
+        public Vector2 CustomScaler
+        {
+            get
+            {
+                Vector2 outVal = Vector2.One;
+                Vector2 size = base.Size;
+                Debug.Log(size);
```

Comment:
There is a stray `Debug.Log(size)` here and another one a few lines below

Reply:
Ye it's not done and doesn't work 100%

Reply:
Ah, fair enough. You could mark this pull request as a draft then.